tests/tui/test_falff.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+
+from functools import partial
+from pathlib import Path
+
+from .pilot_functions import run_before_for_reho_falff_preproc
+
+
+def test_falff_at_features_tab(snap_compare, start_app, work_dir_path: Path, downloaded_data_path: Path) -> None:
+    """Adds falff features. All options are modified plus all 'Remove confounds' options are selected"""
+
+    run_before_with_extra_args = partial(
+        run_before_for_reho_falff_preproc,
+        data_path=downloaded_data_path,
+        work_dir_path=work_dir_path,
+        stage="at_features_tab",
+        feature_type="falff",
+    )
+    assert snap_compare(app=start_app, terminal_size=(204, 53), run_before=run_before_with_extra_args)
+
+
+def test_falff_at_spec_preview(snap_compare, start_app, work_dir_path: Path, downloaded_data_path: Path) -> None:
```

Comment:
Is a snapshot test the best way to verify that this works? We could also make the UI save the spec as a JSON file and then compare it to the target JSON. That would also catch all of the spec file that's not in view of the snapshot.